numpy/polynomial/polyutils.py

```diff
@@ -725,6 +726,182 @@ def _fit(vander_f, x, y, deg, rcond=None, full=False, w=None):
     else:
         return c
 
+def _fitnd(vandernd_f, coords, data, deg=1, rcond=None, full=False, w=None,
+           max_degree=None):
+    """A simple 2D polynomial fit to data x, y, z
+    The polynomial can be evaluated with
+    numpy.polynomial.polynomial.polyval2d
+
+    Parameters
+    ----------
+    vander2d_f : {function(array_like, ..., int) -> ndarray, list of function(array_like, int) -> ndarray}
+        The 2d vander function, such as ``polyvander2d``,
+        or a list of 1d vander functions for each dimension
```

Comment:
Can we remove the complexity here and only allow the list of 1d vander functions?

Reply:
Sure, will change that soon